### PR TITLE
Jobs payload fix

### PR DESCRIPTION
**Pull request by isaac73**
Fix related to this [issue](https://github.com/Azure/azure-iot-sdk-java/issues/327)

### CODE REVIEW

**Comment by isaac73** [comment]
https://github.com/Azure/remote-monitoring-services-java/issues/42

**Comment by isaac73** [comment]
https://github.com/Azure/azure-iot-pcs-remote-monitoring-java/issues/59

**Comment by isaac73** [comment]
https://github.com/Azure/azure-iot-pcs-remote-monitoring-dotnet/issues/52

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java [hunk 1]
```diff
@@ -99,20 +101,35 @@ public Jobs(final IIoTHubWrapper ioTHubService,
...
-        long maxExecutionTimeInSeconds)
-        throws ExternalDependencyException {
+            String jobId,
+            String queryCondition,
+            MethodParameterServiceModel parameter,
+            Date startTime,
+            long maxExecutionTimeInSeconds)
+            throws ExternalDependencyException, InvalidInputException {
+
+        Map<String, Object> mapPayload = new Hashtable<String, Object>();
```
Comment (molly-moen) [inline iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java]: can you add a comment explaining why this is needed?

Comment (isaac73) [reply iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java]: done

File: iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java [hunk 2]
```diff
@@ -99,20 +101,35 @@ public Jobs(final IIoTHubWrapper ioTHubService,
...
-        MethodParameterServiceModel parameter,
-        Date startTime,
-        long maxExecutionTimeInSeconds)
-        throws ExternalDependencyException {
+            String jobId,
+            String queryCondition,
+            MethodParameterServiceModel parameter,
+            Date startTime,
+            long maxExecutionTimeInSeconds)
+            throws ExternalDependencyException, InvalidInputException {
```
Comment (ppathan) [inline iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java]: nit: Why were the tabs changed?

Comment (isaac73) [reply iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/services/Jobs.java]: Fixed in https://github.com/Azure/remote-monitoring-services-java/pull/50

**Pull request merged**